packages/build/src/plugins/list.js

```diff
@@ -45,6 +45,8 @@ const fetchPluginsList = async function ({ logs, pluginsListUrl }) {
 }
 
 const PLUGINS_LIST_URL = 'https://netlify-plugins.netlify.app/plugins.json'
+// 10 seconds HTTP request timeout
+const PLUGINS_LIST_TIMEOUT = 1e4
```

Comment:
why not make it `10 * 1000`? feels weird to make the code so hard to understand

Reply:
I do not have a strong preference either way.
However, whichever way, I do think it should be consistent with the rest of the codebase, which currently use this format.